Lib/test/test_builtin.py

```diff
@@ -0,0 +1,2044 @@
+# Python test set -- built-in functions
+
+import ast
+import builtins
+import collections
+# import decimal XXX RustPython
```

Comment:
Decimal was added recently in #1789, you could probably uncomment this if you want.